### DISCUSSION TITLE
Getting edge coords from Sobel or Canny

### DISCUSSION THREAD

**Discussion by FNGgames** [libvips/libvips#3524] [Q&A]
Is it possible to get the pixel coordinates of detected edges from an image? For example, getting the coords of the non-zero pixels from a Sobel or Canny image? 

I'm trying to do contour tracing essentially, so trying to get an ordered list of edge point coordinates.

**Comment by jcupitt** [comment]
There isn't a thing to trace paths to arrays of points, unfortunately :( 

There is some stuff for finding connected areas and computing stats: https://github.com/libvips/libvips/discussions/3523
